edgecast/auth/auth_provider.go

```diff
@@ -1,4 +1,5 @@
-// Copyright Edgecast, Licensed under the terms of the Apache 2.0 license. See LICENSE file in project root for terms.
+// Copyright 2021 Edgecast Inc. Licensed under the terms of the Apache 2.0 license.
```

Comment:
Should it be : Copyright 2021 Edgecast Inc., Licensed under the terms of the Apache 2.0 license.
i.e. a comma after Copyright 2021 Edgecast Inc.

Reply:
Updated, thank you!